libraries/AP_DDS/dds_xrce_profile.xml

```diff
@@ -30,6 +30,13 @@
       </historyQos>
     </topic>
   </data_writer>
+  <data_reader profile_name="time__dr">
+    <topic>
+      <kind>NO_KEY</kind>
+      <name>rt/ROS2_Time</name>
+      <dataType>builtin_interfaces::msg::dds_::Time_</dataType>
+    </topic>
+  </data_reader>
```

Comment:
Maybe remove? The example reader is only implemented  for `msgs/Joy`. If we did want to synchronise with an external clock we should use `msgs/Clock` on `rt/ap/clock`.

Reply:
Agree

Reply:
Have removed the said configurations